sandbox-lincbrain.tfvars

```diff
@@ -0,0 +1,11 @@
+singleuser_image_repo = "dandiarchive/dandihub"
+region = "us-west-1"
+singleuser_image_tag = "latest"
+jupyterhub_domain = "hub-sandbox.lincbrain.org"
+jupyterhub_name = "sandbox-lincbrain"
+dandi_api_domain = "https://staging-api.lincbrain.org"
+admin_users = ["asmacdo", "dandibot", "satra", "aaronkanzer", "kabilar"]
+github_client_id = "fffb5efe40b01617b837"
+github_client_secret = "<something-i-am-storing-locally>"
+danditoken = "0ead6c0e8fad218d9beb795c2889bc597a0220a8"
```

Comment:
this should also be env var i think

Reply:
I added this just to resolve.

https://github.com/lincbrain/linc-hub/pull/1/files#r1561742254

Reply:
Just to note for security sake: I renewed the token as well FYI, so that one is expired